internal/api/v1/user/accesskeys/access_key_api.go

```diff
@@ -13,12 +13,14 @@ import (
 	"github.com/go-resty/resty/v2"
 )
 
-type AccessKeyAPI struct {
+const route = "api/v1/users/current"
```

Comment:
I think you can add here `/current/keys` and in this way avoid joining path with keys in every method?

Reply:
Not quite, because at the lower layer, the url.URL argument is passed with only the base URL set (for ex. _http://localhost:3000_ what stands for default SPV Wallet API address ). No prefixes are included—each prefix is added individually within the internal structure of the API package. This approach helps mitigate previous migration challenges, allowing any API to transition to a new route independently.